presto-kudu/src/main/java/com/facebook/presto/kudu/properties/KuduTableProperties.java

```diff
@@ -132,11 +131,6 @@ public KuduTableProperties(TypeManager typeManager)
                         "Number of tablet replicas. Uses default value from Kudu master if not specified.",
                         null,
                         false),
-                stringProperty(
```

Comment:
Instead of adding in one commit and then removing in very next commit, can you please not to add this property at all?

Reply:
Table property `num_replicas` renamed to `number_of_replicas` and squashed all changes related to the table properties.